src/vrapp/VolumeVisualizationApp.cpp

```diff
@@ -511,14 +513,14 @@ void VolumeVisualizationApp::onRenderGraphicsContext(const VRGraphicsState& rend
       std::cout << "Error initializing GLEW." << std::endl;
     }
 #endif        
-
+std::cout << "init vizapp " << std::endl;
```

Comment:
consider logging library

Reply:
this one might do the trick:

  https://github.com/gabime/spdlog

we can put it as an additional issue